.github/workflows/coffee.yml

```diff
@@ -19,22 +19,18 @@ jobs:
       - name: Setup Go environment
         uses: actions/setup-go@v5
 
-      - name: Run Test and Send Slack Message
-        run: go run main.go coffee-break
-        env:
-          SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
-          HACBS_CHANNEL_ID: ${{ secrets.HACBS_CHANNEL_ID }}
```

Comment:
probably a matter of opinion, but since running the command and pushing the changes are 2 different things, I thought it makes sense to keep them separated...

Reply:
Done